### PR TITLE
Update README.md - FIT doc, ANT SDK link

### PR DESCRIPTION
**Pull request by xmedeko**
IMO the "Major Changes From Original Version" should be moved to Wiki or some changelog or at least shortened.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,9 +59,8 @@ Major Changes From Original Version
 
 Updating to new FIT SDK versions
 --------------------------------
-Download the latest SDK from http://www.thisisant.com/pages/products/fit-sdk
 
-Update the profile:
+Download the latest [ANT FIT SDK](http://www.thisisant.com/resources/fit) and update the profile:
```
Comment (pR0Ps) [inline README.md]: I like that you updated the link but I think the formatting was better before when the two steps were separate.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
-http://www.thisisant.com/pages/products/fit-sdk.
+FIT protocol
+------------
+
+See the [ANT FIT doc](https://www.thisisant.com/developer/ant/ant-fs-and-fit1) - scroll down to the *GETTING FAMILIAR WITH FIT FORMAT* chapter. Download [ANT FIT SDK](http://www.thisisant.com/resources/fit) zip and read the `D0000*.pdf` documents from it, start with the `D0000* Flexible & Interoperable Data Transfer (FIT) Protocol*.pdf`. The FIT file structure is defined in the `Profile.xlsx`.
 
 Major Changes From Original Version
 -----------------------------------
 
+After a few years of laying dormant we are back to
```
Comment (pR0Ps) [inline README.md]: To be consistent with the rest of the doc this should be wrapped to 80 chars

File: README.md [hunk 3]
```diff
@@ -1,19 +1,25 @@
 python-fitparse
 ===============
 
-Here's a Python library to parse ANT/Garmin `.FIT` files.
+Here's a Python library to parse ANT/Garmin `.FIT` files. The FIT (Flexible and Interoperable Data Transfer) file protocol is specified by [ANT](http://www.thisisant.com/).
```
Comment (pR0Ps) [inline README.md]: This added bit belongs down in the "FIT Protocol" section

File: README.md [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
+pip install fitparse
+```
 
-The FIT (Flexible and Interoperable Data Transfer) file protocol is specified by
-ANT (http://www.thisisant.com/) and an SDK is available for download at
-http://www.thisisant.com/pages/products/fit-sdk.
+FIT protocol
+------------
+
+See the [ANT FIT doc](https://www.thisisant.com/developer/ant/ant-fs-and-fit1) - scroll down to the *GETTING FAMILIAR WITH FIT FORMAT* chapter. Download [ANT FIT SDK](http://www.thisisant.com/resources/fit) zip and read the `D0000*.pdf` documents from it, start with the `D0000* Flexible & Interoperable Data Transfer (FIT) Protocol*.pdf`. The FIT file structure is defined in the `Profile.xlsx`.
```
Comment (pR0Ps) [inline README.md]: This is too detailed for a README file. Maybe something like:
```
FIT Files
---------
- FIT files contain data stored in a binary file format.
- The FIT (Flexible and Interoperable Data Transfer) file protocol is specified by
  [ANT](http://www.thisisant.com/)
- The SDK, code examples, and detailed documentation can be found in the
  [FIT SDK](http://www.thisisant.com/resources/fit)
```

**Comment by xmedeko** [comment]
@pR0Ps thanks for the review. What about the "Major Changes From Original Version" move to the [releases tab](https://github.com/dtcooper/python-fitparse/releases)?

**Comment by pR0Ps** [comment]
@xmedeko Personally, I have no issue with doing that. @dtcooper is the one that originally put it in the README though so I would ping him first before taking it out.

**Comment by xmedeko** [comment]
Changed as requested (hopefully).
I have looked at the docs. Maybe it should be updated, too. I may try it, if you like.

**Pull request merged**